### ISSUE TITLE
ARG linting falsely requires optional default value

### ISSUE THREAD

**Issue by mulmschneider** [RedCoolBeans/dockerlint#45]
### Issue details

ARGs without a default value create an 'invalid format' error (see below), even though they are perfectly valid according to the [docker reference docs](https://docs.docker.com/engine/reference/builder/#arg).

Also, the same error message is printed three times.
### Steps to reproduce/test case
~~~
$ cat /tmp/Dockerfile 
FROM busybox
ARG user1
ARG buildno
~~~
~~~
$ make js && coffee bin/dockerlint.coffee -f /tmp/Dockerfile 
npm install
coffee -c bin/dockerlint.coffee
coffee -o lib -c src
# Insert shebang so the resulting script runs standalone
{ echo '#!/usr/bin/env node '; cat bin/dockerlint.js; } > bin/dockerlint.js.tmp
mv bin/dockerlint.js.tmp bin/dockerlint.js
ERROR: ARG invalid format user1 on line 2
ERROR: ARG invalid format user1 on line 2
ERROR: ARG invalid format user1 on line 2

ERROR: /tmp/Dockerfile failed.
~~~

### Affected version(s)

master

**Comment by rnagy** [comment]
Fixed in d5a3c0461d31a770d5eb76b98bf7a9332b0a0945

**Comment by EvgenyOrekhov** [comment]
I can still reproduce this issue using the latest [redcoolbeans/dockerlint](https://hub.docker.com/r/redcoolbeans/dockerlint/) Docker image.

Dockerfile:

```dockerfile
FROM busybox
ARG user1
ARG buildno
```

`docker run --rm -t -v "$PWD/Dockerfile:/Dockerfile:ro" redcoolbeans/dockerlint`:

```text
ERROR: ARG invalid format user1 on line 2

ERROR: /Dockerfile failed.
```
